app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str, health: int = 100, hidden: bool = False
+    ):
```

Comment:
```suggestion
    def __init__(
            self, 
            name: str, 
            health: int = 100, 
            hidden: bool = False
    ) -> None:
```
Тут None в init кудись пропало 🙂